src/app/search/items/search-filters/sockets.ts

```diff
@@ -337,6 +338,35 @@ const socketFilters: ItemFilterDefinition[] = [
       return (_item) => false;
     },
   },
+  {
+    keywords: 'readytoenhance',
+    description: tl('Filter.ReadyToEnhance'),
+    destinyVersion: 2,
+    filter: () => (item) => {
+      if (!item.crafted || !item.craftedInfo) {
+        return false;
+      }
+      const level = item.craftedInfo.level;
+      if (item.crafted === 'enhanced') {
+        return (
+          (level >= 11 && item.craftedInfo.enhancementTier < 2) ||
+          (level >= 17 && item.craftedInfo.enhancementTier < 3)
```

Comment:
Is it worth tying these directly to the enhancement socket rather than the craftedInfo numbers?